Snowdog_Components/components/Templates/contact-us/_contact-us.scss

```diff
@@ -20,7 +20,7 @@
     }
 
     &__description,
-    &__description > p {
+    &__description p {
```

Comment:
optionally: shouldn't we change it only in case of EE with PAge Builder? Perhaps for normal version we can still use  `> p` ? I guess usually we won't have different styles for p element inside description element so it's not crucial but perhaps some project have some more nested levels of p and they don' t use page builder.

Reply:
I think it's not necessary to change only for EE. I've updated this to have separate styles for `&__description` and `&__description p`.